docs/CHANGELOG.md

```diff
@@ -78,6 +79,8 @@ extra bounds from `AnimatablePrefab` and `AnimationSetPrefab` ([#1435])
 * `NetConnection` is automatically created when client starts sends data to server. ([#1539])
 * User will receive `NetEvent::Connected` on new connection and `NetEvent::Dissconnected` on disconnect. ([#1539])
 * Added a `pivot` field to `UiTransform`. ([#1571])
+* Fix fly_camera example initial camera and cube position. ([#1582])
+* Add to fly_camera example code to release and capture back mouse input, and to show and hide cursor. ([#1582])
```

Comment:
please also add the link to the #1582 pr. you can see how it is done for other pull requests a bit lower in the same file.

Reply:
Okay!